.gitignore

```diff
@@ -29,3 +29,8 @@ build
 
 #VScode
 .vscode
+
+# Carthage
+Carthage/Checkouts
+Carthage/Build
+*.zip
```

Comment:
This would block all ZIP files everywhere, which would be a very suprising feature.

If Carthage has to store ZIP files somewhere in the project, then please prefix the path for it.